src/metabase/cmd.clj

```diff
@@ -133,33 +133,17 @@
         (for [[k v] (partition 2 args)]
           [(mbql.u/normalize-token (subs k 2)) v])))
 
-(defn- resolve-enterprise-command [symb]
-  (try
-    (classloader/require (symbol (namespace symb)))
-    (resolve symb)
-    (catch Throwable e
-      (throw (ex-info (trs "The ''{0}'' command is only available in Metabase Enterprise Edition." (name symb))
-                      {:command symb}
-                      e)))))
-
-;;; [[load]] and [[dump]] are the SerDes v1 versions. [[import]] and [[export]] are SerDes v2 versions of [[load]]
-;;; and [[dump]]. The v1 versions are deprecated.
-
-(defn- import*
-  "Impl for [[load]] (v1) and [[import]] (v2)."
-  ([path v2?]
-   (import* path v2? "--mode" :skip "--on-error" :continue))
```

Comment:
One major issue with this approach is that if the command is invoked with any option, you lose _both_ defaults, so these are only defaults if you don't specify _any_ options at all.